### PR TITLE
[LWG 6] P2918R2 Runtime format strings II

### LINKED ISSUE #6664: [2023-11 LWG Motion 6] Runtime format strings II

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #6664.
Fixes cplusplus/papers#1598

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -15528,6 +15528,17 @@
   template<class charT, class... Args>
     struct basic_format_string;
 
+  template<class charT> struct @\exposid{runtime-format-string}@ {                  // \expos
+  private:
+    basic_string_view<charT> @\exposid{str}@;                                       // \expos
+  public:
+    @\exposid{runtime-format-string}@(basic_string_view<charT> s) noexcept : @\exposid{str}@(s) {}
+    @\exposid{runtime-format-string}@(const @\exposid{runtime-format-string}@&) = delete;
+    const @\exposid{runtime-format-string}@& operator=(const @\exposid{runtime-format-string}@&) = delete;
```
Comment (mordante) [inline source/utilities.tex:15537]: While implementing this paper I noticed the `const` qualifier for the return type. Can it be removed editorial?

Comment (jensmaurer) [reply source/utilities.tex:15537]: In general, no, but this is a deleted function, so it doesn't matter.

@burblebee , could you please remove the "const" here (in a separate commit)?

Comment (mordante) [reply source/utilities.tex:15537]: Thanks! (I wouldn't have asked if the function wasn't deleted.)

**Review by JohelEGP — APPROVED** [review]

File: source/utilities.tex (hunk 1 above)
Comment (burblebee) [reply source/utilities.tex:15537]: done :)

**Pull request merged**